website/docs/blocks.md

```diff
@@ -127,10 +133,24 @@ JSON-RPC
 }
 ```
 
+```
+JSON-RPC
+{
+    "jsonrpc": "2.0",
+    "id": "1",
+    "method": "devnet_abortBlocks",
+    "params": {
+        "starting_block_id": BLOCK_ID
+    }
+}
+```
+
 Response:
 
 ```
 {
     "aborted": [BLOCK_HASH_0, BLOCK_HASH_1, ...]
 }
 ```
+
+If both `starting_block_hash` and `starting_block_id` are passed, error is thrown. Note: param `starting_block_hash` is being depreciated as block hash is contained in `BLOCK_ID` object.
```

Comment:
It's "deprecated", not "depreciated". But actually, do we not want to make a breaking change and have just `starting_block_id` (no `starting_block_hash`)?

cc: @mikiw

Reply:
Fine with me, just let me know what you decide.

Reply:
Unfortunately, we need to handle that as a breaking change since the whole type has changed and also functionality itself.

Reply:
> starting_block_id

let's stay with `starting_block_id`